index.html

```diff
@@ -0,0 +1,129 @@
+
+            <div class="menu">
+                <i class="fa-brands fa-facebook-f"></i>
+                <i class="fa-brands fa-twitter"></i>
+                
+                <ul class="header-menu">
+                    <li class="header-menu-item"><a href="#work">English</a></li>
+                    <li class="header-menu-item"><a href="#about">My Page</a></li>
+                    <li class="header-menu-item"><a href="#contact">Login</a></li>
+                </ul>
+            </div> 
+        </nav> 
+        <nav class="navbar-white">
+            <div>
+                <img src="images/vg-logo.png" alt="VG Logo">
+            </div>
+            <ul class="header-white-menu">
+                <li class="header-white-menu-item"><a href="#about">About</a></li>
+                <li class="header-white-menu-item"><a href="#program">Program</a></li>
+                <li class="header-white-menu-item"><a href="#join">Join</a></li>
+                <li class="header-white-menu-item"><a href="#sponsor">Sponsor</a></li>
+                <li class="header-white-menu-item"><a href="#news">News</a></li>
+            </ul>
+            <button class="white-button" type="submit">CC Campaign</button>
+        </nav>  
+    </header>
+
+    <main>
+        <section class="headline">
+            <h1 class="headline-hello">Achievers world</h1>
+            <h2 class="headline-summit"> VIBRANT AFRICANS'<br>YOUTH SUMMIT 2022</h2>
+            <p class="headline-para">A global network of professional Software Developers 
+                aimed at sharing opportunities and creating a positive change 
+                with people from over 80 countries is taking place in Nigeria.
+            </p>
+            <p class="headline-venue">@Eko Hotel and Suites, Victoria Island, Lagos, Nigeria.</p>
+        </section>
+
+        <section class="program" id="program">
+            <h1 class="main-program">Main Program</h1>
+            <hr class="line">
+            <div class="program-container">
+                <!--Dynamically Generated-->
+            </div>            
+            <button type="button" class="join-button" id="join">Join Viable Group GS 2022</button>
+            <h3 class="see-program">SEE THE WHOLE PROGRAM</h3>
+        </section>
+
+        <section class="speakers">
+            <h1 class="header-speakers">Featured Speakers</h1>
+            <hr class="line">
+
+            <div class="speakers-container">
+                <!--Dynamically Generated-->
+            </div>
+        </section>
+
+        <section class="about" id="about">
+            <h1 class="headline-hello">Achievers world</h1>
+            <h2 class="about-summit"> VIBRANT AFRICANS' YOUTH SUMMIT 2022</h2>
+            <p class="about-para">A global network of professional Software Developers 
+                aimed at sharing opportunities and creating a positive change 
+                with people from over 80 countries is taking place in Nigeria.
```

Comment:
- [ ]  Based on project requirements it is necessary to use font families `Cocogoose, Lato`  as provided in the theme:
<img width="1045" alt="Screenshot 2022-04-13 at 17 45 41" src="https://user-images.githubusercontent.com/77038610/163194544-10e000d2-c657-4017-9205-5b11253cc1d8.png">